onnxruntime/core/providers/cuda/tensor/pad.cc

```diff
@@ -149,7 +209,6 @@ Status Pad<T>::ComputeInternal(OpKernelContext* ctx) const {
       input_dims,
       input_strides,
       lower_pads,
-      upper_pads,
```

Comment:
Very minor perf optimization for the "generic" implementation code path. Upper pads is redundant info and is actually never used by the kernel. Removing this saves the copy to the device incurred due to passing this by value